Dockerfile

```diff
@@ -0,0 +1,75 @@
+################################################################################
+# Base image
+# Based on https://github.com/andrewmclagan/nginx-hhvm-docker
+################################################################################
+
+FROM nginx
+
+################################################################################
+# Add HHVM repo
+################################################################################
+
+RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
+RUN apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0x5a16e7281be7a449 && \
+    echo deb http://dl.hhvm.com/debian jessie main | tee /etc/apt/sources.list.d/hhvm.list
+
+################################################################################
+# Install supervisor, HHVM & tools
+################################################################################
+
+RUN apt-get install debian-archive-keyring
+RUN apt-get update -o Retries=25
+RUN apt-get install -my -o Retries=25 \
+	hhvm \
+	wget \
+	sqlite3 \
+	supervisor \
+    && apt-get clean
+
+################################################################################
+# Install tools
+################################################################################
+
+RUN cd $HOME && \
+    wget http://getcomposer.org/composer.phar && \
+    chmod +x composer.phar && \
+    mv composer.phar /usr/local/bin/composer && \
+    wget https://phar.phpunit.de/phpunit.phar && \
+    chmod +x phpunit.phar && \
+    mv phpunit.phar /usr/local/bin/phpunit  
+
+RUN composer global require hirak/prestissimo
+
+################################################################################
+# Configuration
+##############################################################################
+
+COPY ./docker/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
```

Comment:
The alternative being:
- The proxy container
- The nginx container
- The PHP 7 php-fpm  container, with SQLite support

And all of that tied together with `docker-compose`